package.json

```diff
@@ -67,12 +66,17 @@
     "typescript": "^3.9.7"
   },
   "dependencies": {
-    "@carbon/icons-react": "^10.14.0",
+    "@carbon/icons-react": "^10.48.0",
+    "@tiptap/extension-document": "^2.0.0-beta.15",
```

Comment:
do we have to use beta releases?

Reply:
Unfortunately, TipTap v1 isn’t supported anymore (https://tiptap.dev/overview/upgrade-guide) and there is no not beta official version https://www.npmjs.com/package/@tiptap/core